Updates to cli docs to current state (#2373)
shopware/docs: guides/development/start-developing.md

- `custom/` — your plugins and themes
- `bin/console` — application CLI (Symfony console), runnable from your host via `shopware-cli project console`
- The Administration UI

> ## Running Commands

Reviewer: ```suggestion
## Running commands
```

> ## Frontend Development

Reviewer: ```suggestion
## Frontend development
```

> When developing the Administration or Storefront, use watchers for Hot Module Replacement. Start them directly from the DevTUI Overview tab (key `1`), or from the command line:

Reviewer: ```suggestion
When developing the Administration or Storefront, use watchers for Hot Module Replacement. Start them directly from the development TUI Overview tab (key `1`), or from the command line:
```
